platform-plugins.md

```diff
@@ -158,36 +158,56 @@ This creates a plugin project in the `hello/` folder with the following content:
 * `example/`:
    - A Flutter app that depends on the plugin, and illustrates how to use it.
 
-The plugin files can be edited in their "natural" IDEs, once the plugin
-project has been built once by Flutter tooling.
+### Editing plugin source code {#edit-code}
 
-For Android, execute
-```
-cd hello/example
-flutter build apk
-```
-Now you can import the Android project in Android Studio from the welcome screen,
-or File->New->Import Project... in the menu. Point Android Studio to the
-`hello/example/android/build.gradle` file. Accept the Gradle Sync dialog, but decline to update
-the Gradle plugin, if asked. Android Studio now imports and builds the project using Gradle.
-When done, the Project view has the plugin code in `hello/java/` and the example app code in
-`app/java/`. You can now ask Android Studio to run the example app in the usual way, by pressing
-the &#9654; button.
-
-For iOS, execute
-```
-cd hello/example
-flutter build ios
-```
-Ignore the error about provisioning profiles. Using Xcode, open `hello/example/ios/Runner.xcworkspace`.
-The Project navigator shows the plugin code in `Pods/Development Pods/hello/Classes/` and the
-example app code in `Runner/Runner/`. You can now ask Xcode to run the example app in the usual way, by
-pressing the &#9654; button.
-
-For the Dart code, you can directly open `hello/` in IntelliJ IDEA to edit the plugin code, or
-`hello/example/` to edit the example app code. In both cases, the relevant Dart code is in the `lib`
-folder in the Project view. Once again, the example app can be run from within IDEA in the
-usual way.
+#### Plugin API code (.dart)
+
+To edit the Dart plugin API code, open `hello/` in IntelliJ IDEA (or your
+favorite Dart editor), The plugin API is located in `lib/main.dart` shown in the 
+Project view.
+
+To run the plugin, you need to launch the plugin example app, which requires
+defining a launch configuration:
+
+1. Select 'Run > Edit Configurations...'.
+1. Select '+' and then 'Flutter'.
+1. In 'Dart entrypoint', enter `<plugin folder>/example/lib/main.dart`.
+1. Select 'OK'.
+1. Launch the example app with 'Run' or 'Debug'.
+
+#### Android platform code (.java)
```

Comment:
(not introduced by this PR) This is kinda conflicting with what is said here: https://flutter.io/intellij-ide/#edit-android-code